lms/services/blackboard_api.py

```diff
@@ -102,7 +102,13 @@ def request(self, method, path):
 
     def _api_url(self, path):
         """Return the full Blackboard API URL for the given path."""
-        return f"https://{self.blackboard_host}/learn/api/public/v1/{path}"
+
+        if not path.startswith("/"):
+            # Paths that don't start with "/" are treated as relative to this
+            # common Blackboard API path prefix.
+            path = "/learn/api/public/v1/" + path
```

Comment:
This is needed because the `nextPage`'s that Blackboard sends us already have the `"/learn/api/public/v1/"` on them. As a bonus if we even want to use `BlackboardAPIClient` to call an endpoint that doesn't begin with `"/learn/api/public/v1/"` at all we can now do so: just start the path with a `"/"`

Reply:
An alternative to this is to accept a `url` as well as a `path` and only run the path through `_api_url` if `url` is missing.